velox/exec/tests/SpillTest.cpp

```diff
@@ -153,11 +158,14 @@ class SpillTest : public testing::Test,
 TEST_F(SpillTest, spillState) {
   // Set the target file size to a large value to avoid new file creation
   // triggered by batch write.
-  spillStateTest(1'000'000'000, 2, 10, 2 * 10);
+  spillStateTest(1'000'000'000, 2, 10, 1, 2 * 10);
+  spillStateTest(1'000'000'000, 2, 10, 10, 2 * 10);
```

Comment:
nit: Perhaps, add a comment clarifying which call tests with duplicates and which without.